### DISCUSSION TITLE
Remember which Runelite profile depending on the account

### DISCUSSION THREAD

**Discussion by hejhej233** [runelite/runelite#16568] [Ideas]
If I'm using a jagex account and I have 2 accounts, I want runelite to remember which profile to remember if I'm on Bob1 (where I would have profile 1) and bob2 etc.

..As well as remembering which screen I had the runelite client on before closing so it opens up at the same one again.

**Comment by Nightfirecat** [comment]
We are working on creating a system to associate RuneLite profiles with ingame accounts (which would also let you associate profiles to individual server profiles such as Leagues, Deadman Mode, Quest Speedrunning, etc.), but it is not complete yet.
